keyboards/handwired/dactyl_manuform/dmote/62key/keymaps/default/keymap.c

```diff
@@ -0,0 +1,152 @@
+#include "62key.h"
+#include "rgblight.h"
+#include <keymap_colemak.h>
+#include <sendstring_colemak.h>
+
+extern keymap_config_t keymap_config;
+
+#define MODS_SHIFT_MASK (MOD_BIT(KC_LSHIFT) | MOD_BIT(KC_RSHIFT))
+#define MODS_CTRL_MASK (MOD_BIT(KC_LCTL) | MOD_BIT(KC_RCTRL))
+#define MODS_ALT_MASK (MOD_BIT(KC_RALT) | MOD_BIT(KC_LALT))
+
+// Layer ID:
+#define _QWERTY 0   // OS-side Colemak. Default.
```

Comment:
## Lines 13-15:

Non-consecutive layer indexes will increase your firmware size for no change in functionality. We recommend using an `enum` for defining the layer indexes.

```c
enum layer_names {
    _QWERTY,   // OS-side Colemak. Default.
    _COLEMAK,  // Keyboard-side Colemak. Portability, emergency.
    _NUMERIC
};
```

This will change your `_NUMERIC` layer's index to `2`.